python/README.md

```diff
@@ -45,5 +47,5 @@ ipython kernel install --user --name=ohsome-examples
 
 Run jupyter notebook.
 ```bash
-jupyter notebook
+venv/bin/jupyter notebook jupyter-notebooks
```

Comment:
is the `venv/bin/` here really necessary? when I tested it, after activating the _venv_ it already used the correct `jupyter` command: 

```bash
$ rm -r venv
$ python3 -m venv venv
$ source venv/bin/activate
$ pip install -r requirements.txt
$ which jupyter
/…/ohsome-examples/python/venv/bin/jupyter
```